src/common/Types.ts

```diff
@@ -976,6 +976,8 @@ export interface NavigatorProps extends CommonProps {
     transitionCompleted?: () => void;
     cardStyle?: ViewStyleRuleSet;
     children?: ReactNode;
+    initialRouteStack?: Array<NavigatorRoute>;
```

Comment:
In lieu of supporting the `initialRoute` property you're example code uses the following logic.
It appears this example would work for the `initialRouteStack` property as well.
```
componentDidMount() {
        // Now that the app is mounted, specify the initial
        // navigator route.
        this._navigator.immediatelyResetRouteStack([{
            routeId: NavigationRouteId.MainPanel,
            sceneConfigType: RX.Types.NavigatorSceneConfigType.Fade
        }]);
}
```